File: backend/core/core-domain/src/main/java/com/ssafy/taskit/domain/Sprint.java
package com.ssafy.taskit.domain;

import java.time.LocalDate;

public record Sprint(
    Long id,
    String name,
    SprintStatus sprintStatus,
    LocalDate startDate,
    LocalDate dueDate,
    LocalDate completedDate,
    Long projectId) {
  public static Sprint toEmpty() {
    return new Sprint(null, "", null, null, null, null, null);
  }
}

File: frontend/src/shared/model/types/sprint.ts
export enum SprintStatus {
  READY = 'READY',
  ONGOING = 'ONGOING',
  COMPLETED = 'COMPLETED',
}

export interface Sprint {
  id: number;
  name: string;
  sprintStatus: SprintStatus;
  startDate: string | null;
  dueDate: string | null;
  completedDate: string | null;
  cntRemainIssues?: number;
}
